gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionSha256Sum=b586e04868a22fd817c8971330fec37e298f3242eb85c374181b12d637f80302
```

Comment:
Can't the checksum of the binary be kept?

Reply:
I did not remember removing it.  @Siedlerchr is to blame for https://github.com/JabRef/jabref/pull/8433/commits/5d405cb71991c4737d1399e51581297ef6e2f00e
:wink:

Reply:
I just updated gradlew to the latest version, there was somehow a merge conflict or whatever

Reply:
Ok, bit that means the checksum has changed?

Reply:
Refs the discussion here: https://github.com/gradle/gradle/issues/12412#issuecomment-653000921
Apparently, when you don't include the parameter it gets deleted.